fleetctl/stop.go

```diff
@@ -52,12 +52,22 @@ func init() {
 }
 
 func runStopUnit(args []string) (exit int) {
+	if len(args) == 0 {
+		stderr("One unit file must be provided.")
+		return 1
+	}
+
 	units, err := findUnits(args)
 	if err != nil {
 		stderr("%v", err)
 		return 1
 	}
 
+	if len(units) == 0 {
```

Comment:
I don't think we need this check. The fleetctl commands are idempotent, so it is ok to do nothing.